src/store/login/reducers.js

```diff
@@ -0,0 +1,26 @@
+import actions from './actionTypes';
+
+let user = JSON.parse(localStorage.getItem('users'));
```

Comment:
I think this should be updated to JSON.parse(localStorage.getItem('users')); in the sheet.

Reply:
This should actually be user. When you register you add a user to your local storage entry labeled users and when you log in you are checking if the user is in users. If it is then you save their info in user. Happy to hop on a call to explain